src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+- **Debt in borrowed token** - This shows the amount of debt that the user borrowed. The Kensetsu platform allows borrowing of KUSD tokens pegged to USD.
+- **Available to borrow** - Indicates how much a user can borrow until their vault becomes liquidated.
+- **LTV (loan-to-value)** - The debt ratio to the collateral value in your vault, calculated as a percentage.
+- **Stability Fee** - An annual fee on the amount drawn from your vault, calculated as a percentage.
+- **Liquidation penalty** - a fee applied if the value of your collateral falls below a certain threshold, prompting the sale of assets to cover debt. This penalty increases debt.
+
+### Key actions available for Users:
+
+- **Create a vault** — This operation allows users to create a vault in the Kensetsu platform. Users must deposit some collateral tokens to create a vault. Users can not create empty vaults with collateral tokens, but they can create a vault without borrowing tokens.
+
+The Collateral tokens available in Kensetsu are:
+
+- XOR - `0x0200000000000000000000000000000000000000000000000000000000000000`
+- VAL - `0x0200040000000000000000000000000000000000000000000000000000000000`
+- PSWAP - `0x0200050000000000000000000000000000000000000000000000000000000000`
+- TBCD - `0x02000a0000000000000000000000000000000000000000000000000000000000`
+- ETH - `0x0200070000000000000000000000000000000000000000000000000000000000`
+- DAI - `0x0200060000000000000000000000000000000000000000000000000000000000`
+
+* **Borrowing** - Users borrow KUSD tokens, which are pegged to value of USD. The amount of tokens available to borrow is calculated based on the risk parameters of the Kensetsu platform.
+  Users can borrow tokens in existing vaults if the tokens available to borrow are more than zero.
+
+* **RepayDebt** - Users can repay their borrowed token debt, making the vault healthier and preventing its liquidation. To repay their debt, users must have enough of the borrowed token to repay all fees.
+
+* **Liquidation** - Liquidation is the process where the vault sells the collateral token. Kensetsu will liquidate some of the collateral tokens until the LTV (loan-to-value) metric becomes less than the liquidation threshold.
+
+::: info
+How to avoid a vault’s liquidation?
+
+- **Borrowing tokens with less than 50 % LTV** — It will take you enough time to pay the stability fee and protect yourself in case the collateral token fluctuates.
+- **Add collateral** - Kensetsu allows the deposit of additional collateral tokens, which leads to a decreasing LTV.
+  :::
+
+### How is the stability fee utilised (buyback, burn, farming)
+
+The stability fee that will be charged from these vaults will be used in a stability fund with several purposes:
+
+- **Stabilising KUSD** - By buying back and burning KUSD tokens if the KUSD price goes under $1 (repeg).
```

Comment:
Stability fee is collected in KUSD, and thus it cannot be directly used to buy back KUSD. It can be sold if price of KUSD > $1 to have another token in stability fund and then buy back KUSD, it can be added to LP and it can be used to incentivization. Also stability fund covers bad debt in case of unsuccessful liquidation.